frontend/src/components/cluster-dashboard/cluster-list-view/ClusterListView.tsx

```diff
@@ -0,0 +1,158 @@
+          props.refreshClusterData();
+        }
+      })
+      .catch(function (error) {
+        console.log(error);
+        setSnackBarMessage(error.message && error.message.message ? error.message.message : error.message);
+        setOpenSnackBar(true);
+      })
+  }
+
+  const handleMemberConfirm = ({ member, mode }: { member: string; mode: string }): void => {
+    setOpenConfDialog(false);
+    axios.put(akkaGetClusterMembersUrl + "/" + member.split('://')[1], 
+        querystring.stringify({ operation: mode }),
+        { headers: { "Content-Type": "application/x-www-form-urlencoded" }}
+      )
+      .then(response => {
+        if(response.status === 200) {
+          setSnackBarMessage(response.data.message);
+          setOpenSnackBar(true);
+          props.refreshClusterData();
+        }
+      })
+      .catch(function (error) {
+        console.log(error);
+        setSnackBarMessage(error.message && error.message.message ? error.message.message : error.message);
+        setOpenSnackBar(true);
+      })
+  }
+
+  return (
+    <React.Fragment>
+      <Grid item xs={3} className="shukra-left-container pos-rel">
+        <div className="member-list-container">
+          <div className="member-list-container-title">
+            CLUSTER NODES
+          </div>
+          <TreeView className="member-tree-view" defaultCollapseIcon={<ArrowDropDownIcon />} defaultExpandIcon={<ArrowRightIcon />}>
+          {
+            clusterData && clusterData.members ? clusterData.members.map((member, index) => {
+              return <TreeItem key={index} nodeId={String(index)} label={
+                  <div className="member-header">
+                    <span className="di-inline-blk">
+                      <FiberManualRecordIcon className={"member-status " + member.status.toLowerCase()}/>
+                    </span>
+                    <Tooltip title={member.node} placement="bottom">
+                      <span className="member-name di-inline-blk">{member.node.split('://')[1]}</span>
+                    </Tooltip>
+                    {
+                      member.node === clusterData.leader ? 
+                      <Tooltip title="Leader Node" placement="bottom">
+                        <span className="member-type di-inline-blk leader">L</span>
+                      </Tooltip>
+                      : ""
+                    }
+                    {
+                        member.node === clusterData.oldest ?
+                        <Tooltip title="Oldest Node" placement="bottom">
+                          <span className="member-type di-inline-blk oldest">O</span>
+                        </Tooltip>
+                        : ""
+                    }
+                  </div>
+                }>
+                  <TreeItem nodeId={index + "child"} label={
+                    <div className="member-details">
+                      <div><span className="fo-we-bo">node : </span><span>{member.node}</span></div>
```

Comment:
I don't like the className "fo-we-bo" because I have no idea what that does.

Reply:
Renamed it to "font-bold". Let me know if that works.